uint8 enableCtrl
uint8 suck
bool isQueued
---
int32 result
uint64 queuedCmdIndex
